app/views/admin/reports/show.html.haml

```diff
@@ -55,10 +55,20 @@
   .report-accounts__item
     - if @report.account.local?
       %h3= t('admin.reports.reported_by')
+      = render 'authorize_follows/card', account: @report.account, admin: true
+      = render 'admin/accounts/card', account: @report.account
     - else
-      %h3= t('admin.reports.reported_by_remote_account')
-    = render 'authorize_follows/card', account: @report.account, admin: true
-    = render 'admin/accounts/card', account: @report.account
+      %h3= t('admin.reports.reported_by_remote_instance')
+      = render 'authorize_follows/card', account: @report.account, admin: true
+      .table-wrapper
+        %table.table
+          %tbody
+            %tr
+              %td= t('admin.reports.created_from_remote_instance')
+              %td= link_to pluralize(@report.from_domain.count, t('admin.accounts.show.report')), admin_reports_path(account_id: @report.account.id)
+            %tr
+              %td= t('admin.reports.targeting_remote_instance')
+              %td= link_to pluralize(@report.targeting_domain.count, t('admin.accounts.show.report')), admin_reports_path(target_account_id: @report.account.id)
```

Comment:
The links still point to reports attributed to / against the particular account and not reports against / reported by users on the instance.

Reply:
Good catch.. this would require changes to ReportFilter to support domains, so you'd end up with a url like `/admin/reports?domain=switter.at` or `/admin/reports?targeted_domain=switter.at`

I'm not sure if that's a good idea.

Reply:
@Gargron would there be any support for a "domains" model which was just the string of the domain + an ID which we could use for joins? It feels a little odd to use the domain in the filter URL — alternatively, I could do the hash of the domain, as prevent the raw domain being in the URL.